SAKPaaS/src/app/components/user-card/user-card.component.scss

```
@@ -14,10 +14,9 @@
     cursor: pointer;
   }
   .settings-icon {
-    display: none;
+    color: var(--midnight-blue);
     float: right;
```

Personally I'd prefer:
```suggestion
    position: absolute;
    right: 0;
```

Just makes more sense imo.

---

You have to add `position: relative;` to `#user-card .content` for it to work though.